Storybook: Add stories for Animate component

mkaz opened the pull request:
## Description

Add stories for Animate component to show in Storybook.

## How has this been tested?

Run: npm run design-system:dev 

Confirm animate component shows in Storybook

mkaz committed:
Components: Add stories for Animate component

epiqueras commented on packages/components/src/animate/stories/index.js:
```diff
@@ -0,0 +1,49 @@
+		{ ( { className } ) => (
+			<Notice className={ className } status="success">
+				<p>Slide-in animation is meant for sidebars and sliding menus.</p>
+				<p>It shows the height and width of the animated element moving from a hidden position to its normal one.</p>
```
I think that we should just focus on what the animation does. The use cases can be a lot more varied than sidebars and sliding menus.

We could also just avoid descriptions altogether for now until we have more thorough copy. The names are very self descriptive as is.

mkaz replied:
Thanks for the reviews, I grabbed the content from the README, wasn't quite sure if we should include within.

epiqueras replied:
What do you think @ItsJonQ?

ItsJonQ replied:
@epiqueras I think some animation examples would be wonderful 😍 . I would be happy to help here you folks need!

> I grabbed the content from the README
@mkaz Unfortunately, the documentation example doesn't really do a lot 😅

epiqueras replied:
Yeah, we need better copy, or we can merge the variations without copy for now.

mkaz replied:
Per this, I've removed the extra documentation and we can improve the README at the same time of what shows for the improved animation examples

epiqueras reviewed (changes requested):
Thanks for working on this :smile:!

mkaz committed:
Update packages/components/src/animate/stories/index.js

Co-Authored-By: Enrique Piqueras <epiqueras@users.noreply.github.com>

mkaz committed:
Update appear options, simplify content

- Updates appear animation with four origin options

- Removes extra text from Notice shown, can update with improved
animation and updated README docs.

- Lint required wrapping quoted text in ``

epiqueras commented on packages/components/src/animate/stories/index.js:
```diff
@@ -0,0 +1,56 @@
+);
+
+// unexported helper for various origins
+const Appear = ( origin ) => (
```
```suggestion
const Appear = ( { origin } ) => (
```

epiqueras commented on packages/components/src/animate/stories/index.js:
```diff
@@ -0,0 +1,56 @@
+	</Animate>
+);
+
+export const appearTopLeft = () => Appear( 'top left' );
```
```suggestion
export const appearTopLeft = () => <Appear origin="top left" />
```

epiqueras commented on packages/components/src/animate/stories/index.js:
```diff
@@ -0,0 +1,56 @@
+);
+
+export const appearTopLeft = () => Appear( 'top left' );
+export const appearTopRight = () => Appear( 'top right' );
```
```suggestion
export const appearTopLeft = () => <Appear origin="top left" />
```

epiqueras commented on packages/components/src/animate/stories/index.js:
```diff
@@ -0,0 +1,56 @@
+
+export const appearTopLeft = () => Appear( 'top left' );
+export const appearTopRight = () => Appear( 'top right' );
+export const appearBottomLeft = () => Appear( 'bottom left' );
```
```suggestion
export const appearTopLeft = () => <Appear origin="top left" />
```

epiqueras commented on packages/components/src/animate/stories/index.js:
```diff
@@ -0,0 +1,56 @@
+export const appearTopLeft = () => Appear( 'top left' );
+export const appearTopRight = () => Appear( 'top right' );
+export const appearBottomLeft = () => Appear( 'bottom left' );
+export const appearBottomRight = () => Appear( 'bottom right' );
```
```suggestion
export const appearTopLeft = () => <Appear origin="top left" />
```

mkaz committed:
Apply suggestions from code review

:+1: Components

Co-Authored-By: Enrique Piqueras <epiqueras@users.noreply.github.com>

mkaz committed:
Fix some naming

epiqueras commented on packages/components/src/animate/stories/index.js:
```diff
@@ -0,0 +1,56 @@
+	</Animate>
+);
+
+// unexported helper for various origins
```
```suggestion
// Unexported helper for different origins.
```

epiqueras commented on packages/components/src/animate/stories/index.js:
```diff
@@ -0,0 +1,56 @@
+	>
+		{ ( { className } ) => (
+			<Notice className={ className } status="success">
+				<p>Appear animation. Origin: { origin }</p>
```
```suggestion
				<p>Appear animation. Origin: { origin }.</p>
```

epiqueras committed:
Update packages/components/src/animate/stories/index.js

epiqueras committed:
Update packages/components/src/animate/stories/index.js
